doc/api/cli.md

```diff
@@ -121,6 +121,14 @@ added: v6.0.0
 
 Print stack traces for process warnings (including deprecations).
 
+### `--redirect-warnings`
+<!-- YAML
+added: REPLACEME
+-->
+
+Write process warnings to the given file instead of printing to stderr but
+only if the file does not already exist.
```

Comment:
Just appending works also. Doing so is pretty safe when dealing with multiple node processes since the warning output includes the pid